staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -76,24 +98,6 @@ func WithAuthentication(handler http.Handler, mapper genericapirequest.RequestCo
 	)
 }
 
-func Unauthorized(supportsBasicAuth bool) http.HandlerFunc {
```

Comment:
what about adding the mapper and serializer to this function? Then we can keep the (now failing) unit tests.